docs/releases.md

```diff
@@ -6,11 +6,81 @@ sidebar_label: Releases
+   * Handle the case of multi-attach error sometimes seen on on-premise clusters
+   * A custom resource for Jiva volumes to help with easy access to the volume status
+   
+  For instructions on how to set up and use the Jiva CSI driver, please see. https://github.com/openebs/jiva-operator.
+
+### Key Improvements
+
+- Several bug fixes to the Mayastor volumes along with improvements to the API documentation. See [Mayastor release notes](https://github.com/openebs/Mayastor/releases/tag/v0.7.1). 
+- Enhanced the [NFS Dynamic Provisioner](https://github.com/openebs/dynamic-nfs-provisioner) to support using Cluster IP for dynamically provisioned NFS server. It was observed that on some of the Kubernetes clusters the kubelet or the node trying to mount the NFS volume was unable to resolve the cluster local service. 
+- [ZFS Local PV](https://github.com/openebs/zfs-localpv) added support for resizing of the raw block volumes. 
+- [LVM Local PV](https://github.com/openebs/lvm-localpv) is enhanced with additional features and some key bug fixes like:
+  * Raw block volume support
+  * Snapshot support
+  * Ability to schedule based on the capacity of the volumes provisioned
+  * Ensure that LVM volume create and deletion functions are idempotent
+- NDM partition discovery was updated to fetch the device details from it parent block device.
+
+
+### Key Bug Fixes
+- Fixed an issue with cStor helm chart w.r.t to CVC operator service that was causing velero restores to fail. (https://github.com/openebs/cstor-operators/pull/239)
+- Fixed an issue with cStor CSPC validation causing failures in creating pools with ZFS supported compression algorithms. (https://github.com/openebs/cstor-operators/issues/237)
+- Fixed an issue with the cStor validation webhook not getting deleted in K8s 1.20+ when the OpenEBS namespace is deleted. https://github.com/openebs/openebs/issues/3338)
+- Fixed an issue with Jiva replicas attempting duplicate registration if the Pod IP of the replica changes. (https://github.com/openebs/openebs/issues/3323)
+- Fixed an issue where Jiva cleanup jobs can't be executed when cluster PSP is restrictive. (https://github.com/openebs/openebs/issues/3331)
+- Fixed an issue where cStor pool migration from SPC to CSPC could result in pool failure when migration takes time and the user tries to restart the SPC pool. (https://github.com/openebs/upgrade/issues/82)
+- Fixed an issue with NDM partition discovery causing errors when partitions were present on top of filtered devices like OS disk. (https://github.com/openebs/openebs/issues/3321)
+ 
+
+### Backward Incompatibilities
+
+- Kubernetes 1.17 or higher release is recommended as this release contains the following updates that will not be compatible with older Kubernetes releases. 
+  * The CRD version has been upgraded to `v1`. (Thanks to the efforts from @RealHarshThakur, @prateekpandey14, @akhilerm)
```

Comment:
Should we mention the names of the contributors in this doc? It will be good to mention only in Github release note.